json/src/main/scala/io/odin/json/Formatter.scala

```diff
@@ -0,0 +1,27 @@
+package io.odin.json
+
+import cats.syntax.show._
+import io.circe.syntax._
+import io.circe.Encoder
+import io.odin.LoggerMessage
+import io.odin.formatter.{Formatter => OFormatter}
+import io.odin.formatter.Formatter._
+import perfolation._
+
+object Formatter {
+  implicit val loggerMessageEncoder: Encoder[LoggerMessage] =
+    Encoder.forProduct7("level", "message", "context", "exception", "position", "thread_name", "timestamp")(
+      m =>
+        (
+          m.level.show,
+          m.message(),
+          m.context,
+          m.exception.map(t => formatThrowable(t).toString()),
+          p"${m.position.enclosureName}:${m.position.line}",
+          m.threadName,
+          m.timestamp
```

Comment:
Not really sure if we want the raw timestamp or a nice formatted one?

Reply:
I think the best case is to have ISO 8601 one. Most of the log aggregators should understand it

Reply:
Changed both to proper ISO 8601 ones.